google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/AppEngineCloudType.java

```diff
@@ -202,6 +203,16 @@ public void setName(String name) {
         this.name = name;
       }
 
+      @Nullable
+      @Override
+      public File getFile() {
+        return new File(userSpecifiedFilePath);
```

Comment:
@chanseokoh Maybe I'm not understanding, but couldn't you only show the custom config if the path is set AND is a jar/war (currently the validation ensures that it has a .jar/.war extension)?